### DISCUSSION TITLE
点击“分析句子”出现这个报错

### DISCUSSION THREAD

**Discussion by hidewong** [ZuodaoTech/everyone-can-use-english#993] [General]
Cannot read properties of undefined (reading
'message’)
是什么原因？该怎么解决呢

**Comment by an-lee** [comment]
AI 默认引擎是怎么配置的？

**Comment by hidewong** [comment]
![image](https://github.com/user-attachments/assets/cc149221-35d9-468e-955c-da4902c23cd0)
